web/modules/webspark/webspark_blocks/src/Form/WebsparkBlocksAdminSettings.php

```diff
@@ -58,6 +58,7 @@ public function buildForm(array $form, FormStateInterface $form_state) {
     $form['links_fieldset'] = [
       '#type' => 'fieldset',
       '#title' => $this->t('404 Links'),
+      '#markup' => '<p>When adding links, both the url and title are required.</p>',
```

Comment:
Content, but not the tags, should be wrapped in a `t()` function.

Reply:
Thank you! Added the t() for this property - one note/question, is #markup still valid for form elements in drupal 10? I couldn't find it for reference - I think it'd be #description instead?

Reply:
@chill5-0 Good question. It's still included in [the documentation for Render Arrays in Drupal 10.1](https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Render%21theme.api.php/group/theme_render/10)

Reply:
With or without at t(), according to the docs there, the value is passed through XSS filtering, but I didn't see mention of internationalization, so t() still seems like a safe option, though some of the examples aren't using it.